pkg/handler/enqueue.go

```diff
@@ -77,11 +76,11 @@ func (e *EnqueueRequestForObject) Generic(evt event.GenericEvent, q workqueue.Ra
 }
 
 func request(obj client.Object) reconcile.Request {
-	return reconcile.Request{client.ObjectKey{
-		Cluster: logicalcluster.From(obj),
-		NamespacedName: types.NamespacedName{
+	return reconcile.Request{
+		ClusterName: logicalcluster.From(obj).String(),
```

Comment:
longer-term need to find a way to plumb something that will do this through so we can propose it upstream without depending on kcp